x-pack/plugins/security/index.js

```diff
@@ -75,7 +76,8 @@ export const security = (kibana) => new kibana.Plugin({
       return {
         secureCookies: config.get('xpack.security.secureCookies'),
         sessionTimeout: config.get('xpack.security.sessionTimeout'),
-        rbacEnabled: config.get('xpack.security.rbac.enabled')
+        rbacEnabled: config.get('xpack.security.rbac.enabled'),
+        rbacApplication: config.get('xpack.security.rbac.application'),
```

Comment:
Good call.

Reply:
I chose a more restrictive pattern because it'd look inconsistent to accept punctuation and spaces and then append `_rbac_user` and `_rbac_dashboard_only_user` to the end.